script.js

```diff
@@ -136,3 +137,17 @@ for (let i = 0; i < 6; i += 1) {
     });
   });
 }
+
+// FORM VALIDATION ////////////////////////////////////////////////////////////////////////////////
```

Comment:
[OPTIONAL] Nice work 💯 but instead of comments look for ways to make the code cleaner by splitting it into several parts, basically look to organize the code better, cleaner 🧼